how-to/donate_project.md

```diff
@@ -0,0 +1,111 @@
+# Project Donation Application
+
+Please see the [proposals/new-project-donation-process.md](Documentation) to
+better understand the full process for submitting a new project.
+In short, copy this Application Template and populate the document.
+--!>
+
+## CNCF checklist
+- [ ] All project metadata and resources are vendor-neutral
+- [ ] Governance structure
+- [ ] Contributing guides
+- [ ] Overlap with existing Kubeflow projects
+- [ ] Public list of adopters.
+
+
+## Background information
+
+1. Submitter Name
+    -
+
+1. Submitter’s relationship to project / title
+    -
+
+1. Project Name
+    -
+
+1. Why this project is valuable to the Kubeflow Community
+    -
+
+1. List of integrations with Kubeflow Core components (and potential)
+    -
+
+1. Short Description / Functionality
+    -
+
+1. Website
+    -
+
+1. GitHub repository
+    -
+
+1. 1st Release date
+    -
+
+1. Project Meeting Times
+    -
+
+1. Meeting Notes
+    -
+
+1. Installation Documentation
+    -
+
+1. Project Documentation
+    -
+
+1. Security Profile (CVE scanning)
+    -
+
+1. Ownership / Legal Profile (license type, any potential issues for CNCF)
+    -
+
+1. Authorization, Isolation mechanisms
+    -
+
+1. Project Roadmap
+    -
+
+1. Other Information
+    -
+
+## Metrics
+
+- Number of Maintainers
+- Number of Releases in last 12 months
+- Number of Contributors
+- Number of Users
+- Number of Forks
+- Number of Stars
+- Number of package/project installations/downloads
+
+## Kubeflow Specific Questions
```

Comment:
Should we also say something about conformance ?

Reply:
Any specific suggestions you have?

Reply:
Maybe we can say that if projects want to join the Kubeflow Ecosystem, they should provide at least their initial thoughts or timeline how we they can be included in Kubeflow Conformance program ?

Reply:
I think we need to define conformance first though? Maybe we ask will they commit to conformance once it's defined? That's a little fuzzy but I think reasonable given the ambiguous state.

Reply:
That makes sense, maybe commitment is sufficient initially.
In the future, when we have concrete process, we can update this proposal.

Reply:
Added 👍